src/components/biggive-button/biggive-button.scss

```diff
@@ -96,66 +109,41 @@
     @include button-grey-dark();
 }
 
-.button-clear-primary {
-    @include button-clear-primary();
-}
-
-.button-clear-secondary {
-    @include button-clear-secondary();
-}
-
-.button-clear-tertiary {
-    @include button-clear-tertiary();
-}
-
-.button-clear-brand-1 {
-    @include button-clear-brand-1();
-}
-
-.button-clear-brand-2 {
-    @include button-clear-brand-2();
-}
-
-.button-clear-brand-3 {
-    @include button-clear-brand-3();
-}
-
-.button-clear-brand-4 {
-    @include button-clear-brand-4();
-}
-
-.button-clear-brand-5 {
-    @include button-clear-brand-5();
-}
-
-.button-clear-brand-6 {
-    @include button-clear-brand-6();
-}
-
-.button-clear-white {
-   @include button-clear-white();
-}
-
-.button-clear-black {
-    @include button-clear-black();
- }
-
- .button-clear-grey-extra-light {
-    @include button-clear-grey-extra-light();
- }
- .button-clear-grey-light {
-    @include button-clear-grey-light();
- }
-
- .button-clear-grey-medium {
-    @include button-clear-grey-medium();
- }
-
- .button-clear-grey-dark {
-    @include button-clear-grey-dark();
- }
-
-
 .button-rounded {
     @include button-rounded();
  }
+
+ $colours: (
+  "primary": $colour-primary-blue,
+  "secondary": $colour-secondary-green,
+  "tertiary": $colour-tertiary-coral,
+  "brand-1": $colour-brand-cc-red,
+  "brand-2": $colour-brand-wgmf-purple,
+  "brand-3": $colour-brand-gmf-green,
+  "brand-4": $colour-brand-emf-yellow,
+  "brand-5": $colour-brand-c4c-orange,
+  //"brand-6": $colour-brand-grey,
+  "brand-afa-pink": $colour-brand-afa-pink,
+  "brand-mhf-turquoise": $colour-brand-mhf-turquoise,
+  "white": $colour-white,
+  "black": $colour-black,
+  "grey-extra-light": $colour-grey-extra-light,
+  "grey-light": $colour-grey-light,
+  "grey-medium": $colour-grey-medium,
+  "grey-dark": $colour-grey-dark
+  );
```

Comment:
this var looks good for now - do you think on the next PR it would make sense to move it to somewhere close to https://github.com/thebiggive/components/blob/751dcde1425e8fd674cb0da97f55109913e0a7d8/src/globals/variables.scss#L40 so that it can be shared with the text-colours declared at https://github.com/thebiggive/components/blob/a03da445cb0f4108fa8613a47083023aeccce64e/src/globals/mixins.scss#L1166 that are used in the big give impact block?